administrative/build.gradle

```diff
@@ -7,8 +7,8 @@ repositories {
 }
 
 dependencies {
-    implementation project(":core:configuration")
-    implementation project(":core:client")
+    implementation project(":configuration")
```

Comment:
these changes should be done on the Generator https://github.com/thousandeyes/thousandeyes-sdk-generator/blob/main/thousandeyes-sdk/src/main/resources/thousandeyes-java/build.gradle.mustache

Reply:
Thanks, forgot about that. I've removed those changes from here which is why the build is failing now